.github/workflows/build.yml

```diff
@@ -234,6 +234,14 @@ jobs:
             versionSuffix: 0
             extra_defines: PICO_SUPPORTED
 
+          - name: Zapbox iOS
```

Comment:
Can you rename this to iOS Zapbox, to match the others which is "OS Product" and not "Product OS"?

Reply:
done!